packages/kmath/src/vector.js

```diff
@@ -7,6 +7,8 @@
 import _ from "underscore";
 import * as knumber from "./number.js";
 
+type Vec = $ReadOnlyArray<number>;
```

Comment:
Is it really worth keeping this short rather than just spelling out `Vector`? Or is there a clash with using `Vector`?

Reply:
No clash, I'm used to `Vec` from rust. I'll expand it.